evolver/hardware/standard/od_sensor.py

```diff
@@ -29,5 +32,26 @@ def read(self):
             response = comm.communicate(cmd)
         for vial, raw in enumerate(response.data):
             if vial in self.vials:
-                self.outputs[vial] = self.Output(vial=vial, raw=int(raw))
+                # Calibrate raw data to density.
+                raw = int(raw)
+                density = self._transform("output_transformer", "convert_to", raw, vial)
+                self.outputs[vial] = self.Output(vial=vial, raw=raw, density=density)
         return self.outputs
+
+
+class OD90(ODSensor):
+    class Config(ODSensor.Config):
+        name: str = ODSensor.Config.model_fields["name"].from_field(default="od_90")
```

Comment:
I'm not sure this is correct - I think the intention is to combine the `Field` object information, essentially overriding the default? but this looks like it doesn't do that (`from_field` is a static method)

```py
f = pydantic.Field(default='a', description='test')
f.from_field(default='b')
-> FieldInfo(annotation=NoneType, required=False, default='b')
```

I'm not sure the way this is supposed to be done, but there is a merge field info:
```py
FieldInfo.merge_field_infos(f, FieldInfo(default='b'))
-> FieldInfo(annotation=NoneType, required=False, default='b', description='test')
```

Reply:
Nice catch, I hadn't noticed that, thanks!

Reply:
Rereading the [the docs for this](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.FieldInfo.from_field), I really don't see the point of this method at all. Seems like quite a wasted opportunity and my, assumed functionality, would be far more useful. Never mind...

Reply:
``merge_field_infos`` did the trick, thanks again.